user_docs/en/userGuide.t2t

```diff
@@ -2785,10 +2785,18 @@ You can browse installed incompatible add-ons using the [incompatible add-ons ta
 
 ++ Log Viewer ++[LogViewer]
 The log viewer, found under Tools in the NVDA menu, allows you to view all the logging output that has occurred up until now from when you last started NVDA.
-Using NVDA+F1 will open the log viewer and display developer information about the current navigator object.
 
 Apart from reading the content, you can also Save a copy of the log file, or refresh the viewer so that it shows the most recent output since the Log viewer was opened.
 These actions are available under the viewer's Log menu.
+The file which is displayed when you open the log viewer is saved on your computer in the temporary folder "%temp%" of your Windows and is called NVDA.log.
+This log file is also saved when a new session of NVDA is started or when NVDA crashes. The file is automatically renamaed to NVDA_old.log and a new NVDA log file is generated when the new NVDA session starts.
+
+You can also copy some content of the current log file without opening the log viewer and paste the content from your clipboard anywhere you want.
+%kc:beginInclude
+|| Name | Desktop key | Laptop key | Touch | Description |
+| Open log viewer | nvda+f1 | nvda+f1 | none | this will open the log viewer and display developer information about the current navigator object. |
+| Copy a fragment of the log to the clipboard | ctrl+nvda+shift+f1  | ctrl+nvda+shift+f1 | none | This command pressed once, sets the starting point for the content that should be copied. Pressed a second time after you do an action, sets the end point and places the content between starting point and end point in your clipboard. |

```

Comment:
```suggestion
| Open log viewer | ``NVDA+f1`` | ``NVDA+f1`` | None | This will open the log viewer and display developer information about the current navigator object. |
| Copy a fragment of the log to the clipboard | ``NVDA+control+shift+f1``  | ``NVDA+control+shift+f1`` | None | When this command is pressed once, it sets a starting point for the log content that should be captured. When pressed a second time, it copies the log content since the start point to your clipboard. |
```